proto/beacon/rpc/v1/debug.proto

```diff
@@ -36,6 +35,12 @@ service Debug {
             get: "/eth/v1alpha1/debug/forkchoice"
         };
     }
+    // Returns a list of validators individual vote status of a given epoch.
+    rpc GetIndividualVotes(IndividualVotesRequest) returns (IndividualVotesRespond) {
```

Comment:
Why is this considered a debug method if it is so useful for block explorers?

Reply:
Good question. The conversations I've had with the end consumers, they mentioned that it would be useful, but they didn't mention how they would use it. That's why I'm a bit hesitant to upload to upstream `ethereumapi`

@ppratscher @Buttaa Do you guys want to voice your opnions? debug ok or upstream is preferred

Reply:
I think normal users will also want this to, especially those having live monitoring for their validators, as this is a pretty useful feature. I dont see any big harm in having this upstream, do you guys see any issues with it being non-debug ?

Reply:
I agree with @nisdas that it would be a good idea to have this upstream and not only as a debug method.

Reply:
Thanks for the feedback. Upstream it is then.